### PR TITLE
Update Grant-CsOnlineVoiceRoutingPolicy.md

### PR DESCRIPTION
**Pull request by dariomws**
https://github.com/MicrosoftDocs/office-docs-powershell/issues/5329

### CODE REVIEW

File: skype/skype-ps/skype/Grant-CsOnlineVoiceRoutingPolicy.md [hunk 1]
```diff
@@ -90,7 +97,7 @@ Accept wildcard characters: False
 ```
 
 ### -Global
-When you use this cmdlet without specifying a user identity, the policy applies to all users in your tenant. To skip a warning when you do this operation, specify "-Global".
+When you use this cmdlet without specifying a user identity, the policy applies to all users in your tenant, except any that have an explicit policy assignment. To skip a warning when you do this operation, specify this parameter.
```
Comment (get-itips) [inline skype/skype-ps/skype/Grant-CsOnlineVoiceRoutingPolicy.md:100]:
```suggestion
When you use this cmdlet without specifying a user identity, the policy applies to all users in your tenant, except anyone that have an explicit policy assignment. To skip a warning when you do this operation, specify this parameter.
```

File: skype/skype-ps/skype/Grant-CsOnlineVoiceRoutingPolicy.md [hunk 2]
```diff
@@ -57,6 +57,13 @@ PS C:\> Get-CsOnlineUser | Grant-CsOnlineVoiceRoutingPolicy -PolicyName "Redmond
 
 Example 3 assigns the per-user online voice routing policy RedmondOnlineVoiceRoutingPolicy to all the users in the tenant. To do this, the command first calls the `Get-CsOnlineUser` cmdlet to get all user accounts enabled for Skype for Business Online. Those user accounts are then piped to the `Grant-CsOnlineVoiceRoutingPolicy` cmdlet, which assigns each user the online voice routing policy RedmondOnlineVoiceRoutingPolicy.
 
+### -------------------------- Example 4 --------------------------
+```
+PS C:\> Grant-CsOnlineVoiceRoutingPolicy -PolicyName "RedmondOnlineVoiceRoutingPolicy" -Global
+```
+
+Example 4 assigns the per-user online voice routing policy RedmondOnlineVoiceRoutingPolicy to all the users in the tenant, except any that have an explicit policy assignment.
```
Comment (get-itips) [inline skype/skype-ps/skype/Grant-CsOnlineVoiceRoutingPolicy.md:65]:
```suggestion
Example 4 assigns the per-user online voice routing policy RedmondOnlineVoiceRoutingPolicy to all the users in the tenant, except anyone that have an explicit policy assignment.
```

File: skype/skype-ps/skype/Grant-CsOnlineVoiceRoutingPolicy.md (hunk 1 above)
Comment (tseward) [reply skype/skype-ps/skype/Grant-CsOnlineVoiceRoutingPolicy.md:100]: 'anyone that have' to 'anyone that has'.

File: skype/skype-ps/skype/Grant-CsOnlineVoiceRoutingPolicy.md (hunk 2 above)
Comment (tseward) [reply skype/skype-ps/skype/Grant-CsOnlineVoiceRoutingPolicy.md:65]: 'anyone that have' to 'anyone that has'.

File: skype/skype-ps/skype/Grant-CsOnlineVoiceRoutingPolicy.md (hunk 1 above)
Comment (dariomws) [reply skype/skype-ps/skype/Grant-CsOnlineVoiceRoutingPolicy.md:100]: Are you sure about this update? It's the output of the cmdlet:
image

I think it's related to "any" user, "anyone" user doesn't look good.

**Pull request merged**